azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/AbstractWebAppMojo.java

```diff
@@ -288,7 +284,7 @@ public String getLinuxRuntime() {
 
     public WebContainer getJavaWebContainer() {
         return StringUtils.isEmpty(javaWebContainer) ?
-                WebContainer.TOMCAT_8_5_NEWEST :
+                WebContainer.TOMCAT_85 :
```

Comment:
do we need to clean up the track1 model in this clsas? `com.microsoft.azure.toolkit.lib.appservice.model.*`?

Reply:
it will be finally removed, but not in this PR, orelse compilation fails.